script.video.nfl.gamepass/resources/lib/pigskin/pigskin.py

```diff
@@ -0,0 +1,1166 @@
+"""
```

Comment:
often we request such dependencies to be packaged as script modules and imported as addon dependencies. However since this is really self contained and pretty much only valid for nfl gamepass, its fine to pack it and update it with the addon.

Reply:
Yeah, this lib is only used by this addon.